scc/provider/resource_subaccount.go

```diff
@@ -92,6 +93,9 @@ __Further documentation:__
 							getFormattedValueAsTableRow("`Disconnected`", "The tunnel was previously connected but is now intentionally or unintentionally disconnected."),
 						Optional: true,
 						Computed: true,
+						Validators: []validator.String{
```

Comment:
Could this lead to issues if the computed value is connection failed. Not sure of the lifecycle of the validator i.e. if this gets validated again when the state is stored